src/ui/components/Queue.tsx

```diff
@@ -133,19 +154,19 @@ const fieldComponents = {
           </div>
         )
       default:
-        return '--'
+        return <>--</>
     }
   },
 
-  // FIXME: typings
-  attempts: ({ job }: { job: any }) => job.attempts,
+  attempts: ({ job }) => <>{job.attempts}</>,
 
-  // FIXME: typings
-  delay: ({ job }: { job: any }) =>
-    formatDistanceStrict(job.timestamp + job.delay, Date.now()),
+  delay: ({ job }) => (
+    // REVIEW: What should we do if timestamp comes as null?
```

Comment:
Unsure. Open for suggestions 🙂

Reply:
no delay means it will try to process the job as soon as possible, can be 0 here